driver/src/main/java/com/code_intelligence/jazzer/driver/FuzzTargetRunner.java

```diff
@@ -251,7 +251,7 @@ private static int runOne(long dataPtr, int dataLength) {
       dumpReproducer(data);
     }
 
-    if (Long.compareUnsigned(ignoredTokens.size(), Opt.keepGoing) >= 0) {
+    if (!Opt.dedup || Long.compareUnsigned(ignoredTokens.size(), Opt.keepGoing) >= 0) {
```

Comment:
I explained the situation in which this could occur in more detail in the commit message. It required combining two undocumented features and, with the upcoming changes that found this, was caught by an existing test.